LocalPayment/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
     implementation deps.appCompat
 
     api project(':BraintreeCore')
-    implementation project(':BraintreeDataCollector')
+    implementation project(':PayPalDataCollector')
```

Comment:
same q, re:breaking change

Reply:
We should be good. This main reason we switch to `:PayPalDataCollector` in these other modules is to remove an implicit dependency on Kount.